<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="4" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-03T13:52:54Z" name="Default suite" finished-at="2021-10-03T13:52:54Z" duration-ms="39">
    <groups>
      <group name="All">
        <method signature="GroupsDepends.bGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="bGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
        <method signature="GroupsDepends.dGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="dGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
        <method signature="GroupsDepends.cGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="cGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
      </group> <!-- All -->
      <group name="Regression">
        <method signature="GroupsDepends.cGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="cGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
      </group> <!-- Regression -->
      <group name="Sanity2">
        <method signature="GroupsDepends.bGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="bGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
      </group> <!-- Sanity2 -->
      <group name="Sanity3">
        <method signature="GroupsDepends.dGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" name="dGroupTest" class="com.sk.testngsamples.GroupsDepends"/>
      </group> <!-- Sanity3 -->
    </groups>
    <test started-at="2021-10-03T13:52:54Z" name="Default test" finished-at="2021-10-03T13:52:54Z" duration-ms="39">
      <class name="com.sk.testngsamples.GroupsDepends">
        <test-method signature="bGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" started-at="2021-10-03T13:52:54Z" name="bGroupTest" finished-at="2021-10-03T13:52:54Z" duration-ms="8" status="FAIL">
          <exception class="java.lang.ArithmeticException">
            <message>
              <![CDATA[/ by zero]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArithmeticException: / by zero
at com.sk.testngsamples.GroupsDepends.bGroupTest(GroupsDepends.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArithmeticException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bGroupTest -->
        <test-method is-config="true" signature="afterSomeGroup()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" started-at="2021-10-03T13:52:54Z" name="afterSomeGroup" finished-at="2021-10-03T13:52:54Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSomeGroup -->
        <test-method signature="dGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" started-at="2021-10-03T13:52:54Z" name="dGroupTest" description="Desc of dGroupTest" finished-at="2021-10-03T13:52:54Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dGroupTest -->
        <test-method signature="cGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" started-at="2021-10-03T13:52:54Z" name="cGroupTest" depends-on-groups="Sanity2, Sanity3" finished-at="2021-10-03T13:52:54Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method GroupsDepends.cGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97] depends on not successfully finished methods in group "Sanity2"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method GroupsDepends.cGroupTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97] depends on not successfully finished methods in group "Sanity2"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cGroupTest -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.sk.testngsamples.GroupsDepends@7c83dc97]" started-at="2021-10-03T13:52:54Z" name="afterTest" finished-at="2021-10-03T13:52:54Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.sk.testngsamples.GroupsDepends -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
